nubis/puppet/init.pp

```diff
@@ -29,3 +29,30 @@
 package { 'liblist-compare-perl':
   ensure => present,
 }
+
+class { 'python':
+  version => 'system',
+  pip => true,
+  dev => true,
+}
+
+python::pip { 'aws-cfn-bootstrap':
+  ensure => 'present',
+  require => Class['python'],
+  url => "https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-latest.tar.gz",
```

Comment:
This should probably be pinned to a specific version. It looks like latest right now is

https://s3.amazonaws.com/cloudformation-examples/aws-cfn-bootstrap-1.4-8.tar.gz

Which was updated on the same day and time as the "latest" package.